op-node/flags/flags.go

```diff
@@ -164,3 +163,27 @@ var optionalFlags = append([]cli.Flag{
 
 // Flags contains the list of configuration options available to the binary.
 var Flags = append(requiredFlags, optionalFlags...)
+
+func CheckRequired(ctx *cli.Context) error {
+	l1NodeAddr := ctx.GlobalString(L1NodeAddr.Name)
+	if l1NodeAddr == "" {
+		return fmt.Errorf("flag %s is required", L1NodeAddr.Name)
+	}
+	l2EngineAddr := ctx.GlobalString(L2EngineAddr.Name)
+	if l2EngineAddr == "" {
+		return fmt.Errorf("flag %s is required", L2EngineAddr.Name)
+	}
+	rollupConfig := ctx.GlobalString(RollupConfig.Name)
+	if rollupConfig == "" {
+		return fmt.Errorf("flag %s is required", RollupConfig.Name)
+	}
+	rpcListenAddr := ctx.GlobalString(RPCListenAddr.Name)
+	if rpcListenAddr == "" {
+		return fmt.Errorf("flag %s is required", RPCListenAddr.Name)
+	}
+	rpcListenPort := ctx.GlobalInt(RPCListenPort.Name)
+	if rpcListenPort == 0 {
```

Comment:
```suggestion
	if !ctx.GlobalIsSet(RPCListenPort.Name) {
```
0 is a valid port, to bind to anything available by the system.